umbrella/router.go

```diff
@@ -57,6 +58,20 @@ type funcRoutingInfo struct {
 	Manual      string
 }
 
+// Function to compare a lowercase version of the requested slack command to a lowercase registry command
```

Comment:
Idiomatic Golang documentation requires mentioning the function name in the first word. However, this code is not productionalized yet. So, we'll redo it later.